src/components/Reports/FourteenMonthReports/Layout/Table/TableHead/TableHeadCell/TableHeadCell.tsx

```diff
@@ -30,6 +30,17 @@ const PrintableTableSortLabel = styled(TableSortLabel)(() => ({
   },
 }));
 
+const TableCell = styled(StyledTableCell)(() => ({
+  top: 65,
+  position: 'sticky',
+  '@media print': {
+    position: 'static',
+    top: 0,
```

Comment:
I'm not sure where it is in the code, but could you make the name link black instead of blue when printing? That is how it is on the old angular site.